src/lock-fields-manager.ts

```diff
@@ -16,7 +16,10 @@ export class LockFieldsManager<U extends BaseMongoObject> {
 	constructor(private conf: LockFieldConfiguration) {
 		this.conf.excludedFields = _.union(this.conf.excludedFields, ['objectInfos', '_id']);
 		for (const [key, values] of Object.entries(this.conf.arrayWithReferences ?? {})) {
-			this._arrayWithReferences[key] = _.castArray(values);
+			const valuesAsArray = _.castArray(values);
+			if (LodashReplacerUtils.IS_ARRAY_EMPTY(valuesAsArray)) continue;
```

Comment:
this new line should be covered : https://app.codecov.io/gh/neo9/n9-mongo-client/commit/5036c5e502ebbf64eab48392b80da234d31167f3#133bdafc660af83cc76ec06be078cb6a-R20

Reply:
done!